src/qt/guiutil.cpp

```diff
@@ -762,11 +762,11 @@ QString formatBytes(uint64_t bytes)
     if(bytes < 1024)
         return QString(QObject::tr("%1 B")).arg(bytes);
     if(bytes < 1024 * 1024)
-        return QString(QObject::tr("%1 KB")).arg(bytes / 1024);
+        return QString(QObject::tr("%1 KiB")).arg(bytes / 1024);
```

Comment:
```suggestion
        return QString(QObject::tr("%1 KB")).arg(bytes / 1000);
```

Shouldn't we instead refactor to abide by the IEC 80000-13 standard where `1 KB=1000 bytes (base 10)`? macOS and Unix use this IEC standard, Windows is one of the only systems still treating `1KB=1024 bytes (base 2)`.

Or determine the system type and use its standard. So for Unix/macOS systems use base 10, for Windows use base 2.

Reply:
I would be ok with that. Also with KB/s and change the divisor there accordingly form 1024 to 1000 (see e.g. [Ubuntu units policy](https://wiki.ubuntu.com/UnitsPolicy), which says for network bandwidth base-10 should be used). Whatever approach to solve, having unit KB with divisor 1024 is not correct I think.

Reply:
@wodry yeah, I would say change this to base-10 instead and update the pr/commit description

Reply:
Changed the title to be open for either approach. Would like to see more opinions.